tests/apns-push-service.test.ts

```
@@ -0,0 +1,129 @@
+    },
+  },
+  connect: () => {
+    fakeClient = new FakeClient();
+    return fakeClient;
+  },
+}));
+
+// Import AFTER mock.module so service picks up stubbed http2
+const { ApnsPushService } = await import(
+  "@/api/v2/notifications/apns-push.service"
+);
```

ESLint

[error] Replace `·await·import(⏎··"`@/api/v2/notifications/apns-push.service`"⏎` with `⏎··await·import("`@/api/v2/notifications/apns-push.service`"`

(prettier/prettier)